internal/cli/root/builder_test.go

```diff
@@ -114,3 +119,39 @@ func TestBuilder(t *testing.T) {
 		})
 	}
 }
+
+func TestOutputOpts_printNewVersionAvailable(t *testing.T) {
+	prevVersion := version.Version
+	version.Version = "v1.0.0"
+	defer func() {
+		version.Version = prevVersion
+	}()
+
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockVersionDescriber(ctrl)
+	defer ctrl.Finish()
+
+	mockStore.
+		EXPECT().
+		LatestVersion().
+		Return("v2.0.0", nil).
+		Times(1)
+
+	bufOut := new(bytes.Buffer)
+	opts := &BuilderOpts{
+		store: mockStore,
+	}
+	err := opts.printNewVersionAvailable(bufOut)
+	if err != nil {
+		t.Errorf("printNewVersionAvailable() unexpected error: %v", err)
+		return
+	}
+
+	if got, want := bufOut.String(), `
+A new MongoCLI version is available 'v2.0.0'!
+To upgrade, check https://docs.mongodb.com/mongocli/stable/install/ (run "mongocli config set skip_update_check true" to disable the alert.)
+`; got != want {
+		t.Errorf("printNewVersionAvailable() got = %v, want %v", got, want)
+		return
```

Comment:
```suggestion
		t.Fatalf("printNewVersionAvailable() got = %v, want %v", got, want)
```
but since is the last check just removing return is good enough if you ask me